packages/create-anywidget/index.js

```diff
@@ -81,20 +81,20 @@ let i = 1;
 
 const relative = path.relative(process.cwd(), cwd);
 if (relative !== "") {
-	console.log(`  ${i++}: ${bold(cyan(`cd ${relative}`))}`);
+  console.log(`  ${i++}: ${bold(cyan(`cd ${relative}`))}`);
 }
 
 console.log(`  ${i++}: ${bold(cyan("npm install"))} (or pnpm install, etc)`);
 
 // prettier-ignore
 console.log(
 	`  ${i++}: ${
-		bold(cyan('git init && git add -A && git commit -m "Initial commit"'))
+		bold(cyan('git add -A && git commit -m "add widget infrastructure"'))
```

Comment:
I'd prefer the original message. This makes sense if you're adding to an existing repo, but the previous is for bootstrapping a new repo. I guess we could decide between the two, but for now I'd just prefer to give the instructions for creating a new git repot and commiting.

```suggestion
		bold(cyan('git init && git add -A && git commit -m "Initial commit"'))
```